k8s-intf/src/bolero/support.rs

```diff
@@ -0,0 +1,584 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2025 Open Network Fabric Authors
+
+use bolero::{Driver, TypeGenerator, ValueGenerator};
+use std::net::{Ipv4Addr, Ipv6Addr};
+use std::ops::Bound;
+
+pub fn gen_from_chars<D: Driver>(
+    d: &mut D,
+    chars: &str,
+    min: std::ops::Bound<&usize>,
+    max: std::ops::Bound<&usize>,
+) -> Option<String> {
+    let len = d.gen_usize(min, max)?;
+    (0..len)
+        .map(|_| {
+            chars
+                .chars()
+                .nth(d.gen_usize(Bound::Included(&0), Bound::Excluded(&chars.len()))?)
+        })
+        .collect()
+}
+
+pub struct Ipv4AddrString(pub String);
+
+impl TypeGenerator for Ipv4AddrString {
+    fn generate<D: Driver>(d: &mut D) -> Option<Self> {
+        Some(Ipv4AddrString(
+            Ipv4Addr::from(d.gen_u32(
+                Bound::Included(&0x1000_0000_u32),
+                Bound::Excluded(&0xffff_ffff_u32),
+            )?)
+            .to_string(),
+        ))
+    }
+}
+
+pub struct Ipv6AddrString(pub String);
+
+impl TypeGenerator for Ipv6AddrString {
+    fn generate<D: Driver>(d: &mut D) -> Option<Self> {
+        Some(Ipv6AddrString(
+            Ipv6Addr::from(d.gen_u128(
+                Bound::Included(&0x0000_0000_0000_0000_0000_0000_0000_0001_u128),
+                Bound::Excluded(&0xffff_ffff_ffff_ffff_ffff_ffff_ffff_ffff_u128),
```

Comment:
also Included

Reply:
Removed, this is not used by the value generators

Reply:
Code removed.